src/cli/README.md

```diff
@@ -330,6 +331,25 @@ Set the Thread Leader Weight.
 Done
 ```
 
+### leaderpartitionid
+
+Get the Thread Leader Partition ID.
+
+```bash
+> leaderpartitionid
+2147483647
+Done
+```
```

Comment:
if partition id is not set by cli, here, it will return the initial value (0). I think it should be ok to indicate the fact no specific partition id is set by CLI so far.